mkt/search/indexers.py

```diff
@@ -408,14 +412,29 @@ def attach_trending_and_popularity_mappings(cls, mapping):
 
     @classmethod
     def extract_popularity_trending_boost(cls, obj):
+        def get_dict(obj, prop):
+            qs = getattr(obj, prop).filter(
+                region__in=MATURE_REGION_IDS + (RESTOFWORLD.id))
+            return dict(qs.values_list('region', 'value'))
```

Comment:
Ah, here's the query code I overlooked. Is it written like this so it will work for both `Webapp` and `Website`?